eng/Directory.Packages.props

```diff
@@ -12,15 +12,15 @@
     <ILAsmPackageVersion>6.0.0-rtm.21518.12</ILAsmPackageVersion>
     <ILDAsmPackageVersion>6.0.0-rtm.21518.12</ILDAsmPackageVersion>
     <MicrosoftILVerificationVersion>7.0.0-alpha.1.22060.1</MicrosoftILVerificationVersion>
-    <MicrosoftTestPlatformVersion>17.5.0</MicrosoftTestPlatformVersion>
+    <MicrosoftTestPlatformVersion>17.10.0</MicrosoftTestPlatformVersion>
     <!--
       VS .NET Runtime
 
       We should try to keep this version in sync with the version of app-local runtime in VS.
     -->
     <MicrosoftNetCoreAppRuntimePackagesVersion>8.0.0</MicrosoftNetCoreAppRuntimePackagesVersion>
     <MicrosoftWindowsDesktopAppRuntimePackagesVersion>8.0.0</MicrosoftWindowsDesktopAppRuntimePackagesVersion>
-    <xunitVersion>2.4.1</xunitVersion>
+    <_xunitVersion>2.9.0</_xunitVersion>
```

Comment:
Why did we need to change this to `_`?

Reply:
Using a `_` prefix is more idiomatic MSBuild code when the property is only used within the same file.